### PR TITLE
Add henrybear327 to etcd team member

### PR DESCRIPTION
**Pull request by henrybear327**
As a team member of the etcd org, I would like to gain triage access to perform my work for the community.

### CODE REVIEW

File: config/etcd-io/sig-etcd/teams.yaml [hunk 1]
```diff
@@ -111,6 +111,7 @@ teams:
     - siyuanfoundation
     - thedtripp
     - tjungblu
+    - henrybear327
```
Comment (ivanvc) [inline config/etcd-io/sig-etcd/teams.yaml:114]: I think the linter will fail because the list is not alphabetically ordered :see_no_evil:

Comment (henrybear327) [reply config/etcd-io/sig-etcd/teams.yaml:114]: Oh no...
Fixing now! Thanks!

**Review by ivanvc — APPROVED** [review]
/lgtm

Thanks, Henry!

**Review by jmhbnz — COMMENTED** [review]
/lgtm
/approve

**Comment by henrybear327** [comment]
Thanks @ivanvc @ahrtr @jmhbnz

**Pull request merged**